src/query/streams.cpp

```diff
@@ -185,7 +186,28 @@ Streams::Streams(InterpreterContext *interpreter_context, std::string bootstrap_
                  std::filesystem::path directory)
     : interpreter_context_(interpreter_context),
       bootstrap_servers_(std::move(bootstrap_servers)),
-      storage_(std::move(directory)) {}
+      storage_(std::move(directory)) {
+  auto set_stream_offset = [ictx = interpreter_context](mgp_list *args, mgp_graph * /*graph*/, mgp_result *result,
+                                                        mgp_memory * /*memory*/) {
+    auto *arg_stream_name = procedure::Call<mgp_value *>(mgp_list_at, args, 0);
+    const auto *stream_name = procedure::Call<const char *>(mgp_value_get_string, arg_stream_name);
+    auto *arg_offset = procedure::Call<mgp_value *>(mgp_list_at, args, 1);
+    const auto offset = procedure::Call<int64_t>(mgp_value_get_int, arg_offset);
+    std::string error = ictx->streams.SetStreamOffset(stream_name, offset);
+    if (!error.empty()) {
+      MG_ASSERT(mgp_result_set_error_msg(result, error.c_str()) == MGP_ERROR_NO_ERROR);
```

Comment:
I think setting the error message is just best effort thing and we should crash memgraph in case it doesn't succeed.

Reply:
Yes, I wanted to say "shouldn't", but my brain was faster than my hands :smile:

Reply:
So here I propose to remove the `MG_ASSERT`.